src/main/java/br/ufpe/cin/mergers/handlers/MethodAndConstructorRenamingAndDeletionHandler.java

```diff
@@ -20,11 +20,19 @@
 import java.util.stream.Collectors;
 
 /**
- * Renaming or deletions conflicts happen when one developer edits a element renamed or deleted by other.
- * Semistructured merge is unable to detect such cases because it matches elements via its identifier, so
- * if a element is renamed or deleted it cannot match the elements anymore. This class overcomes this issue.
+ * Renaming or deletions conflicts happen when one developer edits a element
+ * renamed or deleted by other. Semistructured merge is unable to detect such
+ * cases because it matches elements via its identifier, so if a element is
+ * renamed or deleted it cannot match the elements anymore. This class overcomes
+ * this issue.
+ * 
+ * We classify renaming or deletion in three main cases: (1) Renaming without body
+ * changes: one altered a method' signature only; (2) Renaming with body changes:
+ * one altered the method' signature and body; (3) Deletion: one deleted a method.
```

Comment:
Renaming or deletions conflicts occur when one developer edits an element that was renamed or deleted by another developer. Semistructured merge plain algorithm is unable to detect such cases because it matches elements via their identifiers; so if an element is renamed or deleted, the algorithm cannot match the corresponding elements. This class handles such cases.
  * We classify renaming or deletion in three main cases: (1) Renaming without body
  * changes: one altered a method' signature only; (2) Renaming with body changes:
  * one altered the method' signature and body; (3) Deletion: one deleted a method.

Reply:
@jvcoutinho with relation to "so if an element is renamed or deleted, the algorithm cannot match the corresponding elements", if an element is deleted, we should not expect matching anyway, right? So I'm not sure I understand what you are trying to say.

Reply:
You are right, @pauloborba . ""so if an element is renamed, ..." is the accurate version.